### ISSUE TITLE
Documentation update for length, width, height units in containers for bulk edit and bulk create positions

### ISSUE THREAD

**Issue by mvzhuang** [ArctosDB/arctos#6427]
## Instructions

_This is a template to facilitate communication with the Arctos Code Table Committee. Submit a separate request for each relevant value. This form is appropriate for exploring how data may best be stored, for adding vocabulary, or for updating existing definitions._

_Reviewing documentation before proceeding will result in a more enjoyable experience._

* [Issue Documentation](http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html)
* [Code Table Documentation](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html)
* [Video Tutorial - Submit a Code Table Request](https://youtu.be/t2jHbsRA3lk)

------------------------------

## Initial Request

Want to add that length, width and height units have to be in centimeters because information more difficult to find without hunting around.

_**Proposed Value**: Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

Same values

_**Proposed Definition**: Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

height | no | height of container in centimeters; "0" (no quotes) will update to NULL; blank will be ignored (no updates).
-- | -- | --
length | no | length of container in centimeters; "0" (no quotes) will update to NULL; blank will be ignored (no updates).
width | no |  width of container in centimeters; "0" (no quotes) will update to NULL; blank will be ignored (no updates).

position_width | yes | Width in centimeters of the positions being created 
-- | -- | --
position_length | yes | Length in centimeters of the positions being created
position_height | yes | Height in centimeters of the positions being created

_**Attribute data type** If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)_

_**Attribute controlled values** If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate._

_**Attribute units** If numberical values should be accompanied by units, provide a link to the appropriate units table._

_**Context**: Describe why this new value is necessary and existing values are not._

So that people don't accidentally choose their own units and units are readily available

_**Table**: Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

_**Collection type**: Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

_**Priority**: Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

_**Example Data**: Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request._

_**Available for Public View**: Most data are by default publicly available. Describe any necessary access restrictions._

_**Project**: Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184)._

_**Discussion**: Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary._

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by Jegelewicz** [comment]
These aren't code table items, but they are defined in the way described:

![image](https://github.com/ArctosDB/arctos/assets/5725767/47a1e030-f95a-4f26-a9fe-43f93a739f5d)

Why? Shouldn't these all be accompanied by units so that one could use whatever units they prefer?

**Comment by Jegelewicz** [comment]
@mvzhuang what were you doing where this wasn't clear?

**Comment by Jegelewicz** [comment]
Nevermind - I see it was bulk stuff...

**Comment by Jegelewicz** [comment]
This is not a code table issue - I am closing it in favor of an enhancement request - https://github.com/ArctosDB/arctos/issues/6452

**Comment by dustymc** [comment]
Choosing the wrong template is not grounds for rejection!

**Comment by Jegelewicz** [comment]
I did not reject it - I made a new issue with a more appropriate ask?

**Comment by mvzhuang** [comment]
Oops thanks for fixing the template issue for me. Wasn't sure where to put it.
Yea bulkloaders for container edits and container bulk position creation. The single position creation screen that asks for position dimensions also doesnt have the units.

Was installing the new compactors cabinet barcodes in.

On Jun 22, 2023 4:41 PM, Teresa Mayfield-Meyer ***@***.***> wrote:

EXTERNAL EMAIL: This e-mail is from a sender outside of the UTEP system. Please forward suspicious emails to ***@***.******@***.***> or call 915.747.6324

@mvzhuang<https://github.com/mvzhuang> what were you doing where this wasn't clear?

**Comment by dustymc** [comment]
>made a new issue

??

**Comment by Jegelewicz** [comment]
https://github.com/ArctosDB/arctos/issues/6427#issuecomment-1603404004

**Comment by Jegelewicz** [comment]
I added the height, width, and length descriptions to the instructions on the Load csv page for https://arctos.database.museum/loaders/bulkCreateContainer.cfm?action=ld

and position_length, position_width descriptions to the instructions on the Load csv page for https://arctos.database.museum/tools/bulkCreateContainerPositions.cfm?action=ld
